File: 프로그래머스/unrated/181851. 전국 대회 선발 고사/전국 대회 선발 고사.swift
import Foundation

func solution(_ rank:[Int], _ attendance:[Bool]) -> Int {
    var ans:[Int] = rank
    var temp:[Int] = []
    for i in 0 ..< rank.count {
        if attendance[i] == true {
            temp.append(rank[i])
        }
    }
    
    temp = Array(temp.sorted().prefix(3))
    var abc:[Int] = []
    for i in temp {
        for j in 0 ..< rank.count {
            if rank[j] == i {
                abc.append(j)
                break
            }
        }
    }
    
    
    
    return abc[0] * 10000 + abc[1] * 100 + abc[2]
}

File: 프로그래머스/0/181851. 전국 대회 선발 고사/전국 대회 선발 고사.py
def solution(rank, attendance):
    arr = []
    for i in range(len(attendance)):
        if attendance[i] == True:
            arr.append((i,rank[i]))
        
    arr.sort(key = lambda x : x[1])
    answer = 0
    
    
            
                       
            
    answer = 10000*arr[0][0] + 100*arr[1][0] + arr[2][0]
    return answer